hack/make_metricsdoc.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
nits:

- please consider this shebang instead: `#!/usr/bin/env bash`
- other scripts in the hack folder tend to use dashes in their name instead of underscores